src/contexts/AuthProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { createContext } from 'react';
+
+import useLocalStorageAuth from 'hooks/useLocalStorageAuth';
+import { Auths } from 'types/auth';
+
+type Nullable<T> = T | null;
+
+interface AuthContextType {
+  auth: Nullable<Auths>;
+  setAuth: (authData: Nullable<Auths>) => void;
+}
+
+const AuthContext = createContext<AuthContextType>({
+  auth: null,
+  setAuth: (_authData: Nullable<Auths>) => {
+    // eslint-disable-next-line
+  },
```

Comment:
Can we just:
```suggestion
  setAuth: (_authData: Nullable<Auths>) => { },
```

Reply:
Looks like the eslint doesn't let me do it 🥲
<img width="684" alt="image" src="https://user-images.githubusercontent.com/122268989/221077363-63ee7c9f-a795-4b67-9e24-b1b7025ec06b.png">